cypress/integration/AnnualPlan/APSpecialReportShareOption/reportShareOption.js

```diff
@@ -0,0 +1,29 @@
+/// <reference types="Cypress" />
+
+import { Given, Then } from "cypress-cucumber-preprocessor/steps";
+
+And('the following page options are displayed', dataTable => {
+    for (const { breakpoint, pageOption } of dataTable.hashes()) {
+        if (breakpoint == 'desktop') {
+            cy.get(`#PageOptionsControl1 a[title='${pageOption}']:visible`).should('be.visible');
+        }
+        else {
+            cy.get(`.mobile-page-options #PageOptionsControl1 a[title='${pageOption}']:visible`).should('be.visible');
```

Comment:
since you will be removing the breakpoint from a data table, you can use the following approach we discussed previously :
 ```
 cy.document().then(doc => {
        if (doc.documentElement.clientWidth < breakpoint for tablet) { 
cy.get(`.mobile-page-options #PageOptionsControl1 a[title='${pageOption}']`).should('be.visible');
} else {
cy.get(`#PageOptionsControl1 a[title='${pageOption}']`).should('be.visible');}
})
```

Reply:
fixed it